JS.md

```diff
@@ -199,11 +199,10 @@ function, global, memory and table imports):
     throw a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
   * Otherwise, append `v.[[Table]]` to `imports`.
 
-Let `instance` be the result of evaluating 
-[`Eval.init`](https://github.com/WebAssembly/spec/blob/master/ml-proto/spec/eval.ml#L319)
-with arguments `module` and `imports`.
-Note: this synchronously executes the [`start`](Modules.md#module-start-function)
-function, if present.
+Let `instance` be the result of creating a new
+[`Eval.instance`](https://github.com/WebAssembly/spec/blob/master/ml-proto/spec/eval.ml#L15)
```

Comment:
So this is a bit subtle, but there are two things here: the abstract wasm-spec-defined `Eval.instance` (which probably we'll rename and relink once we have a prose spec in the spec repo) and the JS `WebAssembly.Instance` object we're constructing which "wraps" it.  Here we're making the former; only at the end do we construct the latter.  We create the former here because it is needed to create the exported functions below.  Does that clear it up for you any?